Gemfile

```diff
@@ -18,5 +18,6 @@ group :development do
   gem 'rubocop-rspec'
 end
 
+gem 'puppet', git: 'https://github.com/donoghuc/puppet.git', branch: 'PUP-11002'
```

Comment:
we can point this at 6.x now. My thinking is that by tracking 6.x we can simply re-run the pr tests right before we take up the new gem release. Theoretically it might be worth leaving this pointing to 6.x so we could potentially spot issues before puppet releases.